snforge_std/src/trace.cairo

```diff
@@ -28,3 +29,59 @@ fn get_last_call_trace() -> Array<CallEntryPoint> {
     let mut output = cheatcode::<'get_last_call_trace'>(array![].span());
     Serde::<Array<CallEntryPoint>>::deserialize(ref output).unwrap()
 }
+
+impl DisplayEntryPointType of Display<EntryPointType> {
+    fn fmt(self: @EntryPointType, ref f: Formatter) -> Result<(), Error> {
+        let str: ByteArray = match self {
+            EntryPointType::Constructor => "Constructor",
+            EntryPointType::External => "External",
+            EntryPointType::L1Handler => "L1 Handler",
+        };
+        f.buffer.append(@str);
+        Result::Ok(())
+    }
+}
+
+impl DisplayCallType of Display<CallType> {
+    fn fmt(self: @CallType, ref f: Formatter) -> Result<(), Error> {
+        let str: ByteArray = match self {
+            CallType::Call => "Call",
+            CallType::Delegate => "Delegate",
+        };
+        f.buffer.append(@str);
+        Result::Ok(())
+    }
+}
+
+impl DisplayCallEntryPoint of Display<CallEntryPoint> {
+    fn fmt(self: @CallEntryPoint, ref f: Formatter) -> Result<(), Error> {
+        write!(f, "Entry point type: ")?;
+        Display::fmt(self.entry_point_type, ref f)?;
+        write!(f, "\nSelector: ")?;
+        Display::fmt(self.entry_point_selector, ref f)?;
+        write!(f, "\nCalldata: ")?;
+        Debug::fmt(self.calldata, ref f)?;
```

Comment:
Why are some of them `Debug` and others `Display`? 🤔

Reply:
`Debug` is already implemented in corelib for `ContractAddress` and `Array<T, +Debug<T>>` (so for `Array<felt252>` too), those prints look pretty nice so I reused them

Reply:
Will Display be implemented for those with Debug? Would be nice to use only one trait